### PR TITLE
css: Give the html and body elements the correct display value

### PR DESCRIPTION
**Pull request by robinlinden**

### CODE REVIEW

File: css/default.css [hunk 1]
```diff
@@ -1,3 +1,9 @@
+/* https://html.spec.whatwg.org/multipage/rendering.html#the-page */
```
Comment (GrayHatter) [inline css/default.css:1]: Can we clone these locally somewhere/somehow? I hate depending on 3rd party docs that might vanish, or change.

**Review by GrayHatter — APPROVED** [review]

File: css/default.css (hunk 1 above)
Comment (Zer0-One) [reply css/default.css:1]: Probably wouldn't be hard to add a CI step to make sure that references to specs are valid/live URLs. But I don't think it's a big deal.

Comment (GrayHatter) [reply css/default.css:1]: depending on 3rd party servers to do what you expect is problematic... 

e.g. https://github.com/robinlinden/hastur/pull/462

Comment (robinlinden) [reply css/default.css:1]: I'm with zero-one. Checking links for liveliness would be nice, and I don't think mirroring every 3rd-party thing we link to or depend on would be worth the trouble.

**Review by mkiael — APPROVED** [review]

**Pull request merged**